tensorflow/lite/python/lite.py

```diff
@@ -308,7 +308,8 @@ class TFLiteConverterV2(TFLiteConverterBase):
       to apply when converting the model. E.g. `[Optimize.DEFAULT]`
     representative_dataset: A representative dataset that can be used to
       generate input and output samples for the model. The converter can use the
-      dataset to evaluate different optimizations.
+      dataset to evaluate different optimizations. Note that this is a necessary
```

Comment:
This isn't completely true, this only opts you into int8 post-training quantization. Int8 quantization of weights should still happen without this flag.